src/chunkserver/chunkserver.cpp

```diff
@@ -221,6 +221,11 @@ int ChunkServer::Run(int argc, char** argv) {
     copysetNodeOptions.chunkFilePool = chunkfilePool;
     copysetNodeOptions.localFileSystem = fs;
     copysetNodeOptions.trash = trash_;
+    if (kWalFilePool != nullptr) {
+        FilePoolState_t currentStat = chunkFilePoolPtr_->GetState();
+        uint32_t maxWalSegmentSize = currentStat.chunkSize + currentStat.metaPageSize;
```

Comment:
maxWalSegmentSize shuold be maxWalChunkSize ?

Reply:
@SeanHai 
In chunkserver contxt, we prefer to call it segment which used for WAL. :)